console/console-init/ui/src/context-state-reducer/actions.ts

```diff
@@ -24,6 +24,7 @@ const DELETE_ADDRESS = "DELETE_ADDRESS";
 const PURGE_ADDRESS = "PURGE_ADDRESS";
 const CREATE_ADDRESS = "CREATE_ADDRESS";
 const EDIT_ADDRESS = "EDIT_ADDRESS";
+const LEAVE_CREATE_DEVICE = "LEAVE_CREATE_DEVICE";
```

Comment:
dialog name `LEAVE_CREATE_DEVICE`   don't provide clarity what is purpose this. can you try to revisit name of dialog.